src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+      class="form-search"
+    >
+      <button class="button big-button" type="submit">
+        <img src="images/Search.svg" alt="Search">
+      </button>
+      <input
+        class="input big-input"
+        type="text"
+        name="bigSearch"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+    <form
+    action="#"
+    method="post"
+    data-qa="small"
+    class="form-search"
+    >
+      <button class="button small-button" type="submit">
+        <img src="images/Search.svg" alt="Search">
+      </button>
```

Comment:
Icon shouldn't be button.
Wrap input in label element. 
Create for label pseudo - element with background as icon image.